### PR TITLE
TASK: Cast arguments in TextIterator constructor

### PR DESCRIPTION
**Pull request by dfeyer**

### CODE REVIEW

File: Neos.Utility.Unicode/Classes/TextIterator.php [hunk 1]
```diff
@@ -195,11 +195,11 @@ public function last()
     }
 
     /**
-     * Returns the next elment following the character of the original string
+     * Returns the next element following the character of the original string
      * given by its offset
      *
      * @param integer $offset The offset of the character
-     * @return string The element following this character
+     * @return integer The element following this character
```
Comment (kitsunet) [inline Neos.Utility.Unicode/Classes/TextIterator.php]: `string` was right here? Or either the method name or implementation are wrong. I would expect that this method returns the next character in the subject after the one in offset.

File: Neos.Utility.Unicode/Classes/TextIterator.php [hunk 2]
```diff
@@ -218,7 +218,7 @@ public function following($offset)
      * Returns the element preceding the character of the original string given by its offset
      *
      * @param integer $offset The offset of the character
-     * @return string The element preceding this character
+     * @return integer The element preceding this character
```
Comment (kitsunet) [inline Neos.Utility.Unicode/Classes/TextIterator.php]: Same as above

**Comment by bwaidelich** [comment]
What does this do? ;)

**Comment by kdambekalns** [comment]
This is in master by now via #1018 in more or less the same way. Thanks anyway, @dfeyer!

**Pull request closed without merging**